sources/packages/api/src/services/education-program-offering/education-program-offering.service.models.ts

```diff
@@ -58,3 +59,14 @@ export interface OfferingsFilter {
 export interface PrecedingOfferingSummaryModel {
   applicationsCount: number;
 }
+
+/**
+ * The additional properties which are required
+ * to start the new assessment workflow
+ * and delete the existing workflow instance for an application.
+ */
+export class ApplicationAssessmentSummary extends Application {
+  assessmentWorkflowId: string;
```

Comment:
I missed that before but this one can be null if the application is in submitted status.